pkg/streamformatter/streamformatter_test.go

```diff
@@ -79,5 +82,30 @@ func TestJsonProgressFormatterFormatProgress(t *testing.T) {
 			expectedProgress, expectedProgressShort, msg.ProgressMessage)
 	}
 
-	assert.Equal(t, progress, msg.Progress)
+	assert.Equal(t, jsonProgress, msg.Progress)
+}
+
+func TestJsonProgressFormatterFormatStatus(t *testing.T) {
+	sf := jsonProgressFormatter{}
+	res := sf.formatStatus("ID", "%s%d", "a", 1)
+	assert.Equal(t, "{\"status\":\"a1\",\"id\":\"ID\"}"+streamNewline, string(res))
+}
+
+func TestNewJSONProgressOutput(t *testing.T) {
```

Comment:
This function is named `TestNewJSONProgressOutput`, but it isn't really testing `NewJSONProgressOutput`. `NewJSONProgressOutput` just returns a struct. IMHO it's testing `WriteProgress`.